include/OpenSpaceToolkit/Astrodynamics/Dynamics/Thruster/GuidanceLaw.hpp

```diff
@@ -0,0 +1,103 @@
+/// Apache License 2.0
```

Comment:
I feel like `GuidanceLaw` should be kept outside of `Thruster` 🤔 Not quite sure

Reply:
Yeah I think we should move it under trajectory.

Reply:
Agreed!